app/javascript/components/admin/RoomConfigRow.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import Form from 'react-bootstrap/Form';
+import PropTypes from 'prop-types';
+import {
+  Row, Stack,
+} from 'react-bootstrap';
+
+export default function RoomConfigRow({ title, subtitle }) {
+  return (
+    <Row>
+      <Stack className="my-2" direction="horizontal">
+        <Stack>
+          <strong> {title} </strong>
+          <span className="text-muted"> {subtitle} </span>
+        </Stack>
+        {/* TODO: Refactor this. */}
+        <Form.Select
+          className="visibility-dropdown"
+          // onChange={(event) => {
+          //   updateRoomSetting.mutate({ setting_name: 'muteOnStart', value: event.target.value });
+          // }}
+          // defaultValue={}
```

Comment:
You can remove the comment, or maybe put the Backend and Frontend on same PR?

Reply:
This is going to be changed soon anyways when we start working on it.
I left the commented part since it will need to be used and will help out whoever is working on it (whether it's me or someone else)